game/static/game/css/landing.css

```
@@ -80,9 +80,14 @@ body {
     padding: 1.5rem 5%;
     background-color: var(--nav-bg);
     border-bottom: 1px solid var(--border-color);
+
     position: sticky;
```

Stylelint (17.13.0)

[error] Duplicate property "position" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)